tests/Aspire.Hosting.Elasticsearch.Tests/ElasticsearchFunctionalTests.cs

```diff
@@ -62,23 +62,20 @@ await pipeline.ExecuteAsync(
             }, cts.Token);
     }
 
-    [Theory]
-    [InlineData(true)]
-    [InlineData(false)]
+    [Fact]
+    // [InlineData(true)]
+    // [InlineData(false)]
     [RequiresDocker]
-    public async Task WithDataShouldPersistStateBetweenUsages(bool useVolume)
+    public async Task WithDataShouldPersistStateBetweenUsages()
     {
+        bool useVolume = false;
```

Comment:
Is this just a temporary change? is the true case flaky?

Reply:
Don't look at the source changes in this PR 😄